marketplace_builder/views/pages/how-platformos-works/glossary.liquid

```diff
@@ -77,26 +77,26 @@ Then use yield inside the layout:
 {% endraw %}
 ```
 
-## Custom Attributes
+## Properties
 
-Custom Attributes are fields that you attach to a User Profile, Transactable, Relationship, Custom Model Type, etc.
+Properties are fields that you attach to a User Profile, Transactable, Relationship, Custom Model Type, etc.
 
-Some of them are provided by us to jumpstart your development &ndash; for example, we have a Custom Model Type called `custom_address` and it has multiple Custom attributes baked in.
+Some of them are provided by us to jumpstart your development &ndash; for example, we have a Custom Model Type called `custom_address` and it has multiple Properties baked in.
 
 Learn more:
 
-* [Custom Attributes](/tutorials/custom-attributes/custom-attributes)
+* [Properties](/tutorials/properties/properties)
```

Comment:
This will generate broken link without changing slug, right?
And without changing filename contributions wont work